index.bs

```diff
@@ -1466,9 +1466,86 @@ these steps:
   1. [=/Resolve=] |p| with |result|.
 
 1. Return |p|.
+</div>
```

Comment:
you need a blank line before the </div> for bikeshed to properly realize the markdown formatted ordered list ended.
(i.e. to silence the "FATAL ERROR: Line 1469 isn't indented enough (needs 1 indent) to be valid Markdown:" error)